package.json

```diff
@@ -19,29 +19,31 @@
   ],
   "sideEffects": false,
   "dependencies": {
-    "react": ">=0.14.x",
-    "styled-components": ">=2.4.0"
+    "@babel/runtime": "7.11.2",
```

Comment:
if babel is excluded as dependency, maybe we use `bundled` options instead of `runtime`, according to [rollup-plugin-babel docs](https://www.npmjs.com/package/@rollup/plugin-babel#babelhelpers)?

Reply:
Yes, please.